test/provisioner/provision_aws.go

```diff
@@ -521,6 +521,17 @@ func (v *Vpc) setupSecurityGroup() error {
 					},
 					ToPort: aws.Int32(6443),
 				},
+				{
+					FromPort:   aws.Int32(15150),
```

Comment:
You'll also need to add port 9000 for vxlan, if you want to enable tests which includes pods accessing public network, or testing service exposed by pod (eg. curl <http-service>)

Reply:
Good to know! I didn't enable the e2e tests for aws yet, it should be in part 2 and I will open that port accordingly.